tools/mo/openvino/tools/mo/convert.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Will we have a documentation for this new API?

Reply:
There is a separate ticket for auto-generation of documentation for mo.convert and args parser: 89898